debian/control

```diff
@@ -23,6 +23,7 @@ Build-Depends:
  libsoup2.4-dev,
  libsystemd-dev,
  ostree,
+ ostree-tests (>= 2017.6)
```

Comment:
Sorry, I was thinking the same thing but decided not to nitpick further. Will push a trivial followup.